kit/azure/bootstrap/resources.groups.tf

```diff
@@ -1,14 +1,61 @@
-data "azuread_users" "platform_engineers_members" {
-  # unfortunately mail_nicknames attribute does not work on our AADs because we don't sync from on-premise
-  # so we have to use user prinicpal names for lookups
-  user_principal_names = var.platform_engineers_members[*].upn
-}
-
 resource "azuread_group" "platform_engineers" {
   display_name     = var.platform_engineers_group
   description      = "Privileged Cloud Foundation group. Members have full access to deploy cloud foundation infrastructure and landing zones."
   security_enabled = true
-  owners           = [data.azuread_client_config.current.object_id] # todo: possibly the owner needs to be the cloudfoundation SPN? need to figure out dependency order inside the boostrap module!
+  owners           = [data.azuread_client_config.current.object_id]
+  members          = toset(data.azuread_users.platform_engineers_members.object_ids)
+}
+
+resource "azurerm_role_definition" "cloudfoundation_deploy" {
+  name        = var.platform_engineers_group
```

Comment:
@felixzieger i choose the same name for the role as for the group. do you have any concerns?